glance/files/mitaka/glance-api.conf.Debian

```diff
@@ -799,55 +799,101 @@ stores = {{ server.storage.engine }}
 # (dict value)
 #http_proxy_information =
 
+{%- if 'swift' in storage_engines %}
```

Comment:
I would choose using deeper structure as later you are able to have more than than store under swift.
If there is possibility of having more options under one namespace, we should go deeper than having tons of flat underscored parameters (eg. you can do the same for swift_store:auth).